lib/config/config-file.js

```diff
@@ -472,13 +498,14 @@ function resolve(filePath, relativeTo) {
     let normalizedPackageName;
 
     if (filePath.indexOf("plugin:") === 0) {
+        const pluginPath = filePath;
         const packagePath = filePath.substr(7, filePath.lastIndexOf("/") - 7);
         const configName = filePath.substr(filePath.lastIndexOf("/") + 1, filePath.length - filePath.lastIndexOf("/") - 1);
 
         normalizedPackageName = normalizePackageName(packagePath, "eslint-plugin");
         debug(`Attempting to resolve ${normalizedPackageName}`);
         filePath = resolver.resolve(normalizedPackageName, getLookupPath(relativeTo));
-        return { filePath, configName };
+        return { filePath, configName, pluginPath };
```

Comment:
`pluginPath` is not a very accurate name. Maybe `configFullName` is more appropriate. Something like `plugin:node/recommended`.
`filePath` is modified in the previous line to contain the resolved path of the config file.

I'm not very fond of JSDoc. How should I document the optional properties of a return object?